main.py

```diff
@@ -0,0 +1,157 @@
+import tempfile
+import logging
+from collections import namedtuple
+from datetime import datetime, timedelta
+
+from cryptography import x509
+from cryptography.x509.oid import NameOID
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization, hashes
+from cryptography.hazmat.primitives.asymmetric import rsa
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.addHandler(logging.NullHandler())
+
+CertAttributes = namedtuple('CertAttributes', [
+    'country',
+    'state',
+    'city',
+    'org',
+    'org_name',
+    'common'
+])
```

Comment:
There are two places that you transfer these attributes to `x509.NameAttribute` instances. You can make that a method on this class. You can actually use a `namedtuple` as a class, and add properties methods to it, like this:

```
class CertAttributes(namedtuple(...)):
    def to_x509(self):
        pass
```